plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -17,11 +41,27 @@ pub struct CgroupV2Metric {
     pub time_used_tot: u64,
     pub time_used_user_mode: u64,
     pub time_used_system_mode: u64,
+    pub anon_used_mem: u64,
+    pub file_mem: u64,
+    pub shared_mem: u64,
+    pub file_mapped_mem: u64,
+    pub total_mem: u64
 }
 
 impl FromStr for CgroupV2Metric {
     type Err = anyhow::Error;
-
+    /// # Function
+    /// 
+    /// * `from_str` - Function provides functionality to parse a string by whitespaces places.
```

Comment:
I don't think it is that useful to specify in a function doc that this is a function